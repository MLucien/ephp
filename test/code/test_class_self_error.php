<?php

self::$hello_world = 1;

