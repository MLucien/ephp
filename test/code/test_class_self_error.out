
Fatal error: Cannot access self:: when no class scope is active in {{CWD}}/test/code/test_class_self_error.php on line 3
